client/src/assets/icons/spacer.tsx

```diff
@@ -1,17 +1,21 @@
 import React, { Fragment } from 'react';
 import { useTranslation } from 'react-i18next';
 
-// eslint-disable-next-line
-function Spacer(props: any): JSX.Element {
+interface SpacerPropTypes {
+  style: Record<string, unknown>;
+}
+
+function Spacer(props: SpacerPropTypes): JSX.Element {
   const { t } = useTranslation();
 
+  props.style = { paddingTop: '5', ...props.style };
```

Comment:
Doesn't this change how it's displayed? It's gone from paddingTop to style.paddingTop

Reply:
My understanding is `paddingTop` is not a prop on `svg`, thus needs to be in the `style` attribute.

Reply:
That sounds right, but doesn't that mean these will be padded now?

Reply:
Ah... I will find out.

Reply:
They did already have a `paddingTop` attribute, I just moved it.

Reply:
Oh, not to worry then.  Thanks, Nick.